### ISSUE TITLE
PSA: 7.3 compatibility issues (saving field meta)

### ISSUE THREAD

**Issue by Mosnar** [khalwat/seomatic#380]
This is just a warning and reference for people searching for this issue in the future. One of our servers automatically upgraded to PHP 7.3 due to the version constraint being set incorrectly. Craft compatibility aside, this causes some pretty weird behavior with SEOmatic. The primary symptom I observed was that SEO meta field data would not save (though the overrides were working, no errors thrown).

I suspect this has something to do with string sanitation parameters having changed in 7.3; however, I didn't look into it very much since we weren't supposed to be on 7.3 anyway.

If you're using services like Heroku, make sure you set your PHP version constraint to this to avoid automatic upgrades to 7.3:
` "php": "~7.2.0",`

This package is deprecated and unmaintained, so I'm just posting this to help save the next person some time.

**Comment by darylknight** [comment]
Yep, thanks mosnar. I'm maintaining an old Craft 2 site. My local Valet installation runs on PHP 7.3, and I can't see the front end of this site - getting this error:

    Property "Craft\Seomatic_MetaFieldModel.0" is not defined.

**Comment by ow** [comment]
Ah crap, I'm affected by this too—I haven't been able to migrate yet and this kinda leaves me stranded

**Comment by jamesmacwhite** [comment]
I'm aware this is old issue, but our upstream provider intends to remove PHP 7.2 soon. I deployed our Craft 2 (latest version) site on PHP 7.3.21 to test things and I don't seem to have any issues saving entries and no issues on the front end. No PHP errors related to SEOMatic and it's meta field. Either I'm not hitting the right issue with my environment or this magically got fixed somehow.
